ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -48,23 +79,66 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
         <style>
           :host {
             @apply --layout-vertical;
+            transition: all 0.7s;
           }
           paper-checkbox {
-            --paper-checkbox-label-spacing: 10px;
-            margin-bottom: 5px;
+            width: 100%;
+            margin-top: 6px;
+            --paper-checkbox-label-spacing: 8px;
+          }
+          nuxeo-collapsible {
+            --nuxeo-collapsible-heading-padding: 0;
+          }
+          label {
+            @apply --nuxeo-label;
+          }
+          .showButton a {
+            display: block;
+            margin-top: 4px;
           }
         </style>
 
-        <dom-repeat items="{{buckets}}">
+        <dom-if if="[[_isEmpty]]">
           <template>
-            <paper-checkbox noink checked="{{item.checked}}" on-change="_computeValues">
-              [[item.label]] ([[item.docCount]])
-            </paper-checkbox>
+            <label>[[label]]</label>
+            <span>[[i18n('checkboxAggregation.noResults')]]</span>
           </template>
-        </dom-repeat>
-        <dom-if if="[[_isEmpty]]">
+        </dom-if>
```

Comment:
This means that if I have a nuxeo-checkbox-aggregation with `collapsible` set to `true`:
1) if it is not empty, it shows only the title and you have to click on it to have any more details
2) it is is empty, you will always see the "No available results", which can lead to weird results on a search form.

We need to double check that this is indeed what we want from a UX point of view.